sql/handler.h

```diff
@@ -712,6 +712,7 @@ enum handler_create_zip_dict_result
                                           exists */
   HA_CREATE_ZIP_DICT_NAME_TOO_LONG,  /*!< zip dict name is too long */
   HA_CREATE_ZIP_DICT_DATA_TOO_LONG,  /*!< zip dict data is too long */
+  HA_CREATE_ZIP_DICT_DATA_ZERO_LEN,  /*!< zip dict data is zero len */
```

Comment:
We don't need this. We can check at Server layer and return error from there directly